Sources/GroundState/py_exact.hpp

```diff
@@ -55,29 +56,34 @@ void AddExactModule(py::module &m) {
                matrix_type: The type of matrix used for the Hamiltonian when
                    creating the matrix wrapper. The default is `sparse`. The
                    other choices are `dense` and `direct`.
+               propagation_type: Specifies whether the imaginary or real-time
+                   Schroedinger equation is solved. Should be one of "real" or
+                   "imaginary".
 
            Examples:
-               Solving 1D ising model with imagniary time propagation.
+               Solving 1D Ising model with imaginary time propagation:
 
                ```python
                >>> import netket as nk
                >>> import numpy as np
-               >>> L = 20
+               >>> L = 8
                >>> graph = nk.graph.Hypercube(L, n_dim=1, pbc=True)
                >>> hilbert = nk.hilbert.Spin(graph, 0.5)
                >>> n_states = hilbert.n_states
                >>> hamiltonian = nk.operator.Ising(hilbert, h=1.0)
                >>> stepper = nk.dynamics.create_timestepper(n_states, rel_tol=1e-10, abs_tol=1e-10)
```

Comment:
Alright, since we will address the scipy ode integrator in a later (but just a few months away!) release, we should maybe fix the APIs at this stage. Can we rename `create_timestepper` -> `timestepper` ? It seems to me that the action of creating is implicit in the constructor of the object (we don't have `create_rbm` etc)

Reply:
Ok, I have renamed it.

Reply:
Thanks, great!